_layouts/default.html

```diff
@@ -44,7 +47,7 @@
 
     <footer>
```

Comment:
I _think_ instead of adding `padding` to `<body>`, `<main>` should have `class="container"`, which will give it responsive max-widths. All `rows` need to be in a `container`.